components/avatar/avatar.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Avatar Component</title>
+    <link rel="stylesheet" href="/library/component.css">
+    <style>
+        body {
+            padding: 0px;
+        }
+        .component-holder {
+            display: flex;
+            flex-direction: row;
+            align-items: flex-end;
+        }
+        .comp-lib-hr {
```

Comment:
you can merge two different classes using a comma if they have the same styling

Reply:
Done.